docs/src/docs/extending-testerra/events-and-listener.adoc

```diff
@@ -9,13 +9,12 @@ Testerra provides a Google Guava event bus with custom events for test execution
 |Called on start of every test method annotated by TestNG `@Test` annotation.
 
 |`MethodEndEvent`
-|Called at the end of every test method annotated by TestNG `@Test` annotation. This can be used to send test results to a test management system or issue tracker.
+|Called at the end of every test method annotated by TestNG `@Test` annotation.
 
-//|TEST_START
-//|Called on start of every test method annotated by TestNG `@Test` annotation and every test configuration method annotated by TestNG `@BeforeMethod` or similar, but before the execution of registered <<Before Method Worker>>
-//|TIMESTAMP+
-//ITestResult +
-//IInvokedMethod
+|`TestStatusUpdateEvent`
+|Called after the final result of every test method annotated by TestNG `@Test` annotation. A final result of a test method can be `PASSED`, `RETRIED`, `RECOVERED`, `SKIPPED`, `FAILED` or `EXPECTED_FAILED`.
+
+This event can be used to send test results to a test management system or issue tracker.
```

Comment:
Please add the hint, that this event could be fired multiple times on the same method, because it could be "failed" on the first time and then "retried" on the second.

Reply:
Hm... The tests of TeamCity connector and also my tests with updated AzureDevops connector show only one call per method for this event, if you have a status like `RETRIED`.